### ISSUE TITLE
payer_id is not pulled along with subscriber details when looking for a subscription instance

### ISSUE THREAD

**Issue by thoraxe** [varyonic/paypal-subscriptions-sdk-ruby#1]
I'd like to use the subscription's `payer_id`, and it comes back in the JSON response, but it's not actually put into the returned object:

```ruby
thesub = PayPal::SDK::Subscriptions::Subscription.find("I-UF77YVRH12WE")
thesub.subscriber.name.given_name # joe
thesub.subscriber.payer_id # undefined method
```
I'll try to work up a PR.

**Comment by thoraxe** [comment]
fixed by #2

**Comment by varyonic** [comment]
#2 manually merged.
